samples/javascript/04.live-share-react/src/components/ExampleLiveState.jsx

```diff
@@ -0,0 +1,57 @@
+import { useLiveState } from "@microsoft/live-share-react";
+import { LiveEvent, UserMeetingRole } from "@microsoft/live-share";
+
+const ExampleAppState = {
+    WAITING: "WAITING",
+    START: "START",
+};
+
+const ALLOWED_ROLES = [
+    UserMeetingRole.organizer,
+    UserMeetingRole.presenter,
+    UserMeetingRole.attendee,
+];
+
+export const ExampleLiveState = (props) => {
+    const [state, data, setState] = useLiveState(
+        "CUSTOM-STATE-ID",
+        ALLOWED_ROLES,
```

Comment:
nit: probably better for readability if roles are at end.

Reply:
Done!